pagopa/io-app
# chore: [PE-565] Adapt `CgnCTAStartActivationScreen` to use the new DS components

## Short description
This PR adapts the `CgnCTAStartActivationScreen` to use the new DS components;

## List of changes proposed in this pull request
- Removed the deprecated `BaseScreenComponent` using the new `useHeaderSecondLevel` hook;
- Replaced the deprecated `LoadingErrorComponent` with the use of both `OperationResultScreenContent` if there is an error and the `LoadingScreenContent` if the screen is loading;

## How to test
- Start the CGN onboarding from the Wallet homepage;
- When you tap on the CTA to start the onboarding, you should be able to see for a second a temporary screen that shows a loading indicator spinner;

## Preview
|Before|After|
|-|-|
|<img src="https://github.com/user-attachments/assets/1d6271d7-220f-4c4e-9d86-17ca2aa4435b" width="300px" />|<img src="https://github.com/user-attachments/assets/0487d11f-c350-4177-b268-b7afb1d542de" width="300px" />|
